src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.spec.js

```diff
@@ -240,15 +240,16 @@ describe('The calInboxInvitationMessageBlueBarController', function() {
       expect(ctrl.meeting.loaded).to.equal(true);
     });
 
-    it('should report an invalid meeting if the sequence is outdated', function() {
+    it.only('should not report an invalid meeting if the sequence is outdated , must show an outdated message', function() {
```

Comment:
```suggestion
    it('should not report an invalid meeting if the sequence is outdated and show an outdated message', function() {
```

also you can add another test case instead of modifying this one.